Makefile

```diff
@@ -16,12 +16,11 @@ favicon:
 website:
 	@mkdir -p build && \
 	node_modules/.bin/babel-node frontend/render-badge-examples && \
-	cat frontend/fragments/try-header.html build/badge-examples-fragment.html frontend/fragments/try-footer.html > try.html && \
-	cat try.html | $(SED) "s,(<img src=')(/[^'\?]+)',\1https://img.shields.io\2?maxAge=2592000'," \
+	cat frontend/fragments/try-header.html build/badge-examples-fragment.html frontend/fragments/try-footer.html \
+		| $(SED) "s,(<img src=')(/[^'\?]+)',\1https://img.shields.io\2?maxAge=2592000'," \
 		| $(SED) "s,(<img src=')(/[^'\?]+\?[^']+)',\1https://img.shields.io\2\&maxAge=2592000'," \
 		| $(SED) "s,<span id='imgUrlPrefix'>,&https://img.shields.io," \
```

Comment:
Would be nice to make handle this in code, with a flag that indicates a dev or production build.

Reply:
It's clear that a flag needs to happen before this can be merged. These sed transforms shouldn't happen in development.